src/java/io/bazel/rulesscala/scalac/ScalacProcessor.java

```diff
@@ -87,6 +87,10 @@ public void processRequest(List<String> args) throws Exception {
        */
       copyResources(ops.resourceFiles, ops.resourceStripPrefix, tmpPath);
       /**
+       * Extract and copy resources from resource jars
+       */
+      copyResourceJars(ops.resourceJars, tmpPath);
```

Comment:
Should we copy-in the contents of resource jars before copying in resources, in order to give individual resources precedence?